go.mod

```diff
@@ -14,7 +14,7 @@ require (
 	github.com/gobwas/glob v0.2.3
 	github.com/google/go-github/v32 v32.1.0
 	github.com/kubevirt/monitoring/pkg/metrics/parser v0.0.0-20231024120544-6a3ba1a680b4
-	github.com/machadovilaca/operator-observability v0.0.7
+	github.com/machadovilaca/operator-observability v0.0.19-0.20240326121036-9f2e5a31675f
```

Comment:
I don't think we have a problem with that, but one has to make sure to not take a "half baked" version, so it makes sense for @assafad to explain on the commit message why he didn't choose a formal release (is it because there isn't one yet that has the fix you need? If so - can you try and reach out their maintainer and ask them to issue one?)

Reply:
IIUC it is because real breaking changes that aint easy to fix
https://github.com/kubernetes/client-go/issues/1075#issuecomment-1632677511

CNAO depends on kubevirt client which use old libs, and if we want to update here, it will conflict
but sure lets hear the PR author he knows best of course

Reply:
due to versions incompatibilities between dependency packages, namely `prometheus-operator -> controller-runtime`, the proposed version for the `operator-observability` package, which is not a streamlined version therefore the custom version, uses Golang 1.19 and a downgraded version of `prometheus-operator`  which is compatible with CNAO current state